src/modal/modal.ts

```diff
@@ -0,0 +1,11 @@
+export interface AuthBody {
+  email: string;
+  password: string;
+  name?: string;
+  lastName?: string;
+}
+export interface CategoryType {
+  _id?: string;
+  categoryName: string;
+  img: string;
+}
```

Comment:
why are these types in `modal/modal.ts`, don't make sense to me

Reply:
I create a separate file for the types that I'll be using.
and the name should be types.ts